docs/source/scxml-jani-conversion.rst

```diff
@@ -33,14 +33,68 @@ Low-Level SCXML Conversion
 
 Low-Level SCXML is the standard SCXML format defined `here <https://www.w3.org/TR/scxml/>`_.
 
-Our converter is able to transform high-level SCXML to low-level SCXML by translating the ROS specific features to standard SCXML features.
-In case of timers, we need additional information that cannot be encoded in SCXML, such that information is generated at runtime.
+Our converter is able to transform High-Level (HL) SCXML to Low-Level (LL) SCXML by translating ROS and BT specific features to standard SCXML features.
+This applies also for timers: we generate an additional SCXML FSM encoding a global clock, sending out events to trigger the timers defined in the model at the correct rate.
 
-The conversion between the two SCXML formats is implemented in ScxmlRoot.as_plain_scxml(). TODO: Link to API.
+The next subsections describe our conversion strategy from HL-SCXML to LL-SCXML.
+The entry-point for the conversion is implemented in ScxmlRoot.as_plain_scxml(). TODO: Link to API.
 
-TODO: Describe how we translate the high-level SCXML to the low-level SCXML.
+Handling of (ROS) Timers
+__________________________
+
+TODO
+
+Handling of (ROS) Services
+_____________________________
+
+ROS services, as well as ROS topics, can be handled directly in the ROS to plain SCXML conversion, without the need of adding JANI-specific features, as for the ROS timers.
```

Comment:
In the beginning above we only talk about high and low level SCXML but here also the terminology of plain SCXML is used...

Reply:
I gave it a pass, feel free to look at that again (resolving this comment in the meanwhile)